sdk/identity/identity/src/credentials/azureCliCredential.ts

```diff
@@ -183,4 +180,44 @@ export class AzureCliCredential implements TokenCredential {
       }
     });
   }
+
+  /**
+   * Parses the raw JSON response from the Azure CLI into a usable AccessToken object
+   *
+   * @param rawResponse - The raw JSON response from the Azure CLI
+   * @returns An access token with the expiry time parsed from the raw response
+   *
+   * The expiryTime of the credential's access token, in milliseconds, is calculated as follows:
+   *
+   * When available, expires_on (introduced in Azure CLI v2.54.0) will be preferred. Otherwise falls back to expiresOn.
+   */
+  private parseRawResponse(rawResponse: string): AccessToken {
+    const response: any = JSON.parse(rawResponse);
+    const token = response.accessToken;
+    // if available, expires_on will be a number representing seconds since epoch.
+    // ensure it's a number or NaN
+    let expiresOnTimestamp = Number.parseInt(response.expires_on, 10) * 1000;
+    if (!isNaN(expiresOnTimestamp)) {
+      logger.getToken.info("expires_on is available and is valid, using it");
+      return {
+        token,
+        expiresOnTimestamp,
+      };
+    }
+
+    // fallback to the older expiresOn - an RFC3339 date string
+    expiresOnTimestamp = new Date(response.expiresOn).getTime();
```

Comment:
would it matter if we update this to - (new Date(response.expiresOn)).getTime()

Reply:
@KarishmaGhiya I don't think I understand what you mean, mind helping me out? what would you expect to see changed in the behavior? AFAIK it would not matter but maybe I missed something